bin/init-eg-stack

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash -u
```

Comment:
I feel like maybe there's a little confusion, here. The aim, as far as I understand it, is to support—or rather, make it easier via another method, since extensive work and testing for running on Kubernetes via Helm charts was already done by @LamaAni and @sstubbs (#35 and places)—use of this image on Kubernetes. Except most of this is titled 'stack', whereas usage of `docker stack deploy` has already been supported right from the beginning—just with the Docker Swarm orchestrator:
https://github.com/tiredpixel/postgres-xl-docker/blob/master/docker-compose.swarm.yml
https://github.com/tiredpixel/postgres-xl-docker/blob/master/docker-compose.image.yml
I'd actually expect something very similar to work already… But anyway, it's not that the focus of this is providing Stack support, but rather adding to the Stack support that's already been in place for a long time. Unless I misunderstand something…

Reply:
I've not touched `docker-compose.swarm.yml`, because I don't want to test `swarm` capability.
What about  `docker-compose.image.yml` I don't still know for what it is?

Now I've seen image and swarm file is very same. Maybe we can union these, but it needs the tests.

---

What about https://github.com/tiredpixel/postgres-xl-docker/issues/35 it is very interesting. I've found information about `netcat` as `wait-for-it` in discussion. But I don't know what will be better to use. Tell me what are you thinking about that?

Reply:
Sorry, missed this comment. `image` is for using the pre-built images published to Docker Hub, rather than compiling locally. I'm expecting that most people using this project won't rebuild themselves each time, but rather, will use the published Docker images. However, anyone wanting to develop this project, or indeed to verify the published Docker images, can of course build (like I think you're doing).

As per that discussion, `netcat` was removed and replaced by `pgxc_monitor`. But it turned out to have a use for Kubernetes, so it was readded in https://github.com/tiredpixel/postgres-xl-docker/pull/40 by @LamaAni .